pkgs/development/compilers/elm/packages/release.nix

```diff
@@ -7,5 +7,6 @@
     elm-make = callPackage ./elm-make.nix { };
     elm-reactor = callPackage ./elm-reactor.nix { };
     elm-repl = callPackage ./elm-repl.nix { };
+    elm-format = callPackage ./elm-format.nix { };
```

Comment:
Oh, not nice -- `release.nix` is autogenerated itself.

Let's move this package' call and comments to somewhere inside `elmPkgs` [here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/compilers/elm/default.nix#L65) then...